Cargo.toml

```diff
@@ -117,10 +116,10 @@ getset = "0.1.2"
 hex-literal = "0.4.1"
 
 rand = { version = "0.8.5", default-features = false, features = ["getrandom", "alloc"] }
-warp = { version = "0.3.7", default-features = false }
+axum = { version = "0.7.5", default-features = false }
 wasmtime = "15.0.1"
 
-tracing = "0.1.40"
+tracing = { version = "0.1.40", features = ["log"] }
```

Comment:
Is this an intended change? AFAIK, it enables some adapter subscribers bridging between `log` and `tracing` infrastructure, but I don't see them being used anywhere..

Reply:
it's used in `config/src/lib.rs`, i  think it was silently enabled by `warp`

Reply:
That's right, it was https://github.com/seanmonstar/warp/blob/ce8114b50cadedac089d496235fe9d18596f944e/Cargo.toml#L37.